06-improve-model-performance/improvement-overview.ipynb

```diff
@@ -0,0 +1,913 @@
+{
```

Comment:
<p>Let's separate <code>client = ...</code> and <code>API_KEY = os.environ["CONTEXTUAL_API_KEY"]</code> into their own code block. Reasoning being if someone wants to add a new import they don't have to re-run the client initialization as well.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ContextualAI/examples/pull/40/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>got it, done</p>